RELEASES.md

```diff
@@ -1,3 +1,13 @@
+Version 0.7.4 (2020-03-20)
+========================
+ * New driving model: improved lane changes and lead car detection
+ * Improved driver monitoring model
+ * Improved calibration stability
+ * Improved lateral control on some 2018 and 2019 Toyota Prius
+ * Fixed bug where some 2017 and 2018 Toyota C-HR would use the wrong steering angle sensor
+ * Support for Honda Insight thanks to theantihero!
+ * Code cleanup in car abstraction layers and ui
```

Comment:
Probably worth mentioning 20% more steering torque for VW

Reply:
Will do!